static/js/units-i18n.js

```diff
@@ -26,21 +28,11 @@ export const LOCALE_UNIT_MAP = {
   es: {
     US: IMPERIAL,
     default: METRIC
```

Comment:
would it be possible to get rid of this default too? maybe with like a `LOCALE_UNIT_MAP[language]['default'] || METRIC` on line 61

Reply:
sure